src/bmi_cfe.c

```diff
@@ -99,7 +99,9 @@ Variable var_info[] = {
 	{ 53, "time_step_size",        "int",  1 },
 	{ 54, "is_forcing_from_bmi",   "int",  1 },
 	{ 55, "forcing_file",                                 "string", 1 },  // strlen
-	{ 56, "Schaake_adjusted_magic_constant_by_soil_type", "double", 1 },
+	//{ 56, "Schaake_adjusted_magic_constant_by_soil_type", "double", 1 }, 
```

Comment:
If this variable still exists, but only conditionally, it probably still needs to exist in this list of all possible variables...

Reply:
Done. -for all  `direct_runoff_params_struct` vars
```
    { 85, "Schaake_adjusted_magic_constant_by_soil_type",   "double", 1},
    { 86, "a_Xinanjiang_inflection_point_parameter",        "double", 1},
    { 87, "b_Xinanjiang_shape_parameter",                   "double", 1},
    { 88, "x_Xinanjiang_shape_parameter",                   "double", 1}
```